targets/systemcore/info.arm64.env

```
@@ -0,0 +1,7 @@
+#! /usr/bin/env bash
+
+export TARGET_CPU="cortex-a76"
```

```suggestion
export TARGET_CPU="cortex-a76+crypto"
```
Pi 5 has the crypto extensions